### ISSUE TITLE
Unable to get working ->fillOnCreate or ->fillOnUpdate

### ISSUE THREAD

**Issue by scramatte** [oooholdings/nova-password-generator#22]
Hello

I don't know what I'm doing wroing but I'm unable to get working ->fillOnCreate or ->fillOnUpdate
May some method in my resource interfere with these methods?

BTW Great plugin! You save me lot of work. Thank you

Regards

**Comment by btrsco** [comment]
I'm glad my package has been valuable to you, thanks for the support and kind words!

As for the methods,  I will go through and test this out and try to determine if it's an issue on our end over the weekend. I will have some time then to catch up with some of the issues for this package. I'll update you then!

**Comment by scramatte** [comment]
Any news ?

**Comment by btrsco** [comment]
@scramatte, sorry for the long delay. I have been busier than I'd like to be recently and haven't had time for side projects lately. 

But I have tested and checked whats going on here. This is what I found. I wasn't able to duplicate the issue for `fillOnUpdate()` (seems to be working fine for me on the latest version), though I was able to find that there was a typo when checking for the current form state specifically for `fillOnCreate()`. I have fixed that and have released a fix for that on [v1.8.1](https://github.com/oooholdings/nova-password-generator/releases/tag/v1.8.1)
